.expeditor/config.yml

```diff
@@ -138,6 +142,9 @@ promote:
     - dev
     # acceptance Builder Supervisors update from here
     - acceptance
+    # stable "holding area" for us to perform manual evaluations. This
+    # is only really until we have automated more of our testing.
+    - staging
```

Comment:
Does expeditor use this ordering to promote or is there something baked in to describe the promotion ordering?  I assume the former, but felt it worth asking just in case :wink:

Reply:
Yup, Expeditor uses this list to figure out the promotion order.